src/components/htmlvideoplayer/plugin.js

```diff
@@ -1523,6 +1523,10 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             }
         }
 
+        if (browser.safari || browser.iOS || browser.iPad) {
+            list.push('AirPlay')
```

Comment:
I think we settled on double quotes as new standard, right @dkanada ?

```suggestion
            list.push("AirPlay")
```

Reply:
Nope we will eventually move to single quotes, just haven't changed the old code because it's a huge change.